pkg/plugins/policies/core/matchers/dataplane.go

```diff
@@ -171,6 +174,15 @@ func dppSelectedByPolicy(
 	}
 }
 
+func ddpSelectedByNamespace(meta core_model.ResourceMeta, dpp *core_mesh.DataplaneResource) bool {
+	if meta.GetLabels()[mesh_proto.PolicyRoleLabel] == string(mesh_proto.ConsumerPolicyRole) {
```

Comment:
should be both consumer and workload-owner https://github.com/kumahq/kuma/blob/180976f81dcb6a9b86c4c3fa534e11534060171d/docs/madr/decisions/052-policy-on-namespace.md?plain=1#L103-L102

Reply:
changed